src/views/create-dataset/CreateDataset.tsx

```diff
@@ -0,0 +1,119 @@
+import { ChangeEvent, FormEvent, useEffect, useState } from 'react'
+import { Alert, Button, Col, Form, Row } from '@iqss/dataverse-design-system'
+import { useTranslation } from 'react-i18next'
+import { useDataset } from '../../sections/dataset/DatasetContext'
+import { DatasetSkeleton } from '../../sections/dataset/DatasetSkeleton'
+import { useLoading } from '../../sections/loading/LoadingContext'
+import styles from '/src/sections/dataset/Dataset.module.scss'
+import { FormInputElement } from '@iqss/dataverse-design-system/src/lib/components/form/form-group/form-element/FormInput'
+import { SeparationLine } from '../../components/ui/SeparationLine/SeparationLine'
+import { RequiredFieldText } from '../../components/Forms/RequiredFieldText/RequiredFieldText'
+
+/*
+ * TODO:
+ * Break out different components
+ * Find submit action source
+ * cleanup state objects and submit process
+ * show submitted info in console
+ */
+
+// See comments about https://rjsf-team.github.io/react-jsonschema-form/
+// https://github.com/IQSS/dataverse-frontend/issues/231
+
+export function CreateDataset() {
+  const { setIsLoading } = useLoading()
+  const { isLoading } = useDataset()
+  const { t } = useTranslation('createDataset')
+  const [submitting, setSubmitting] = useState(false)
+  const [submitStatus, setSubmitStatus] = useState(false)
+
+  useEffect(() => {
+    setIsLoading(isLoading)
+  }, [isLoading])
+
+  const [state, setState] = useState({
+    createDatasetTitle: ''
+  })
+
+  const handleCreateDatasetFieldChange = (e: ChangeEvent<FormInputElement>) => {
+    let value: (typeof state)[keyof typeof state] = e.target.value
+    if (e.target.type === 'text') {
+      value = e.target.value
+    }
+
+    setState({ ...state, [e.target.id]: value })
+  }
+
+  // TODO: Connect to API when ready
+  const handleCreateDatasetSubmit = (event: FormEvent<HTMLFormElement>) => {
+    setSubmitting(true)
+    event.preventDefault()
+    setTimeout(() => {
+      setSubmitting(false)
+      console.log('Form Submitted!')
+      setSubmitStatus(true)
+    }, 3000)
+    console.log(state)
+    // TODO: Callback for form submission should be here.
+  }
+
+  // TODO: Probably replace this with a FormSkeleton or remove entirely
+  if (isLoading) {
+    return <DatasetSkeleton />
+  }
+  return (
+    <article>
+      <header className={styles.header}>
+        <h1>{t('pageTitle')}</h1>
+      </header>
+      <SeparationLine />
+      <div className={styles.container}>
+        <RequiredFieldText />
+        {submitting && <div>Submtting Form...</div>}
+        <Form onSubmit={handleCreateDatasetSubmit}>
+          {submitStatus && <div>Form Submitted!</div>}
+          <Row>
+            <Col md={9}>
+              <Form.Group controlId="createDatasetTitle" required>
+                <Form.Group.Label>{t('datasetForm.title')}</Form.Group.Label>
+                <Form.Group.Input
+                  data-cy="datasetFormInputTitle"
```

Comment:
We're using [testing library](https://testing-library.com/docs/) to test from the user's perspective and avoid implementation details, so we're minimizing the use of data-cy as much as possible.

Using testing library the test for this input would be written as `cy.findByLabelText('Title').should('exist').should('have.attr', 'type', 'text')`.